frontend/integration-tests/tests/kubevirt/models/yaml.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars, no-undef */
+import { browser, ExpectedConditions as until } from 'protractor';
+
+import { createItemButton } from '../../../views/crud.view';
+import * as yamlView from '../../../views/kubevirt/yaml.view';
+import { createWithYAMLLink } from '../../../views/kubevirt/wizard.view';
+
+export default class Yaml {
+  async openYamlPage() {
+    await createItemButton.click().then(() => createWithYAMLLink.click());
```

Comment:
can you replace `then` with `await` for better readability (especially when this convection is already used here) ?

Reply:
I think it would be okay, other tests like `crud.scenario.ts` and `secrets.scenario.ts` also has this kind of syntax. 
@rhrazdil WDYT?

Reply:
I'm afraid there is not a clear consistency with ```then``` and ```await``` usage atm, ```then``` syntax is mostly used in views and not in tests.

But it's probably a good idea to try to converge to using ```await``` in all our tests and models for better readability and use ```then``` in views.